src/core/cita_bft.rs

```diff
@@ -62,19 +62,43 @@ pub enum BftTurn {
     Timeout(TimeoutInfo),
 }
 
-#[derive(Debug, PartialEq, PartialOrd, Clone, Copy)]
+#[derive(Debug, Clone, Copy)]
 enum VerifiedBlockStatus {
-    Ok = 1,
-    Failed = -1,
-    Undo = 0,
+    Ok,
```

Comment:
OK?

Reply:
Refer to [std::result::Result]:
```rust
pub enum Result<T, E> {
    Ok(T),
    Err(E),
}
```

[std::result::Result]: https://doc.rust-lang.org/std/result/enum.Result.html